server/base/src/main/java/org/apache/accumulo/server/metadata/iterators/SetEqualityIterator.java

```diff
@@ -79,9 +83,24 @@ public void seek(Range range, Collection<ByteSequence> columnFamilies, boolean i
       int count = 0;
 
       while (source.hasTop()) {
-        byte[] ba = source.getTopKey().getColumnQualifierData().toArray();
-        dos.writeInt(ba.length);
-        dos.write(ba, 0, ba.length);
+        if (includeValue) {
+          byte[] ba = source.getTopKey().getColumnQualifierData().toArray();
+          byte[] valueData = source.getTopValue().get();
+
+          byte[] encodedData = encodeEntry(Map.entry(ba, valueData));
+
+          dos.writeInt(encodedData.length);
+          dos.write(encodedData, 0, encodedData.length);
+        }
+
+        else {
```

Comment:
The build failed because of a checkstyle violation.  If I am reading the error correctly I think it wants the following.

```suggestion
        } else {
```